PostingGuidelines.md

```diff
@@ -0,0 +1,28 @@
+What Should Be Posted
+=====================
+* **DO** post items that would be interesting or useful to the technical / programming community
+* **DO** post original blog posts where you have developed a solution to a problem
+* **DO** post summaries of other blog posts and how they apply to our work - make sure to link to to the source for these
+* **DO** post personal or project achievements if it relates to a particular project or initiative
+* **DO** post links to interesting technologies and articles and explain how we might use them. For these be sure to tag them with "InterestingTech" - Tags coming soon
+* **DO NOT** post internal documentation even if it does not contain sensitive information
```

Comment:
Perhaps worth mentioning/clarifying: where appropriate, project-specific document should be added or improved within the project codebase rather than here